pydocx/export/html.py

```diff
@@ -311,46 +311,46 @@ def export_paragraph_property_justification(self, paragraph, results):
     def export_paragraph_property_indentation(self, paragraph, results):
         # TODO these classes should be applied on the paragraph, and not as
         # inline styles
+
         properties = paragraph.effective_properties
 
         style = {}
 
-        if properties.indentation_right:
-            # TODO would be nice if this integer conversion was handled
-            # implicitly by the model somehow
-            try:
-                right = int(properties.indentation_right)
-            except ValueError:
-                right = None
+        # for numbering properties we add style to span item level
+        if properties.numbering_properties is None:
```

Comment:
How would you feel about switching this conditional so that it was `if properties.numbering_properties` so that the `else` case was the 2-line case? Just a matter of preference, but I struggled for a bit reading this until I realized that the first case was essentially the "do nothing fancy" case.

I'd also love a comment explaining the do-something case like `Numbering properties can define a text indentation on a paragraph`